docs/game_and_server_management/How_to_Get_Your_Game_on_Discord.md

```diff
@@ -65,12 +65,14 @@ Your game looks wonderful. You've got a beautiful store page, and awesome game,
 
 Once you've completed all the items on your checklist, the `Get Approved` button will unlock. Once you submit for approval, Discord will take a look at the following things:
 
-- your game, to ensure that it runs properly
+- your game, to ensure that it runs properly and follows our [Game Submission Guidelines](https://support-dev.discordapp.com/hc/en-us/articles/360025028592-Game-Submission-Guidelines)
 - your store pages, to provide helpful guidance in putting your best foot forward
 - your server, to make sure you're abiding by our [Community Guidelines](https://discordapp.com/guidelines)
 
 If we need to see some changes, you'll receive an email directing you on your steps to success. Otherwise, you'll get **APPROVED!!!!!!!!!!** What's that mean? It means you can start charging for your game and making money! Don't spend it all in one place. Or do. Treat yourself. Or maybe you want your game to be **free!** Live your best life. We support you!
 
+ After that sweet success in being **Approved**, take a moment to celebrate with your community in this grand accomplishment. Now that you feel that *Open Beta* is over on your master branch you can swap from Early Access to now Full Access, not required to be **Approved** mind you! Show off your game to the world freely or with the addition of a price tag to accompany your masterpiece to let those know you have finally evolved! Unleash your next round of attacks at the final boss with the confort of your game in Full Access and the best it can be!  
```

Comment:
I think everything is good with the PR other than this bit; it still reads a little confusing to me. Would it be alright if we just used my suggested edits and then we can merge?

Reply:
> After the sweet success of being **Approved**, take a moment to celebrate with your community. Now, get to work again! You can start charging for your game, or give it away for free. Live your best life. We support you.

Reply:
Yeah sure! Sounds good! Lets me re-sync from Master, remove that bit and should be 100%!

Reply:
Removed! Let me know if this is ready!